### PR TITLE
Better Documentation in README.md

### PR DESCRIPTION
**Pull request by FernandAdam**
#5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
 # 🌉 Discord Bridge
 
-A simple plugin providing a bridge between your discord and minecraft chats.
\ No newline at end of file
+A simple plugin providing a bridge between your discord and minecraft chats.
+
+Partly based on the [CFW](https://github.com/CuukyOfficial/CFW) by Cuuky.
```
Comment (LeStegii) [inline README.md]: The project isn't based on the CFW anymore.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+
+# Dependencies
+- JDK >= 17
```
Comment (LeStegii) [inline README.md]: This project can be compiled with JDK 8 or above.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+# Important Files
+- Main files:
+    - src/net/kettlemc/discordbridge/*DiscordBridge.java*
+        - *Here is the Discord Bridge*
+        - Function:
+          - handle incoming Minecraft and Discord commands
+          - Setup config by methods from Configuration.java
+    - src/net/kettlemc/discordbridge/config/*Configuration.java*
+        - *Here are the important informations such as the **bot key** and the *channel-id* in a java file with a small configuration handler.*
+        - it's based on the configuration manager *GFW*
```
Comment (LeStegii) [inline README.md]: Outdated since it now uses Konfiguration instead of the CFW BasicConfigurationHandler.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+- Minecraft Account
+
+# Important Files
+- Main files:
+    - src/net/kettlemc/discordbridge/*DiscordBridge.java*
+        - *Here is the Discord Bridge*
+        - Function:
+          - handle incoming Minecraft and Discord commands
+          - Setup config by methods from Configuration.java
+    - src/net/kettlemc/discordbridge/config/*Configuration.java*
```
Comment (LeStegii) [inline README.md]: This file doesn't exist anymore since it has been replaced by `DiscordConfig.java`

**Review by LeStegii — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (FernandAdam) [reply README.md]: It didn't work for me. I need JDK 17 or higher.

**Pull request merged**